src/man/meson.build

```diff
@@ -0,0 +1,37 @@
+# The kwarg env: of run_command is only supported by meson>=0.50
+date = run_command(sh, '-c', 'LC_ALL=C date -u +%Y-%b',
```

Comment:
I think this breaks reproducability (IIUC), as the date in the manpage will be set to the build date. Does meson not already have a way to get a reproducible date (i.e. honoring `SOURCE_DATE_EPOCH`)?